emva1288/process/plotting.py

```diff
@@ -409,7 +409,7 @@ class PlotHorizontalSpectrogramDSNU(Emva1288Plot):
 
     name = 'Horizontal spectrogram DSNU'
     xlabel = 'cycles [periods/pixel]'
-    ylabel = 'Standard deviation and\nrelative presence of each cycle [DN]'
+    ylabel = 'Standard deviation and\nrelative presence of each cycle [e^{-}]'
```

Comment:
Once the units are consistent, change this so it formats correctly.
```suggestion
    ylabel = 'Standard deviation and\nrelative presence of each cycle [$e^-$]'
```